### DISCUSSION TITLE
Hyperlocal Marketplace Support

### DISCUSSION THREAD

**Discussion by birender-s** [saleor/saleor#6056] [Q&A] [answered]
I am looking for eCommerce backend platform offering various ecommerce capabilities and also Hyperlocal Marketplace feature exposed via REST APIs to be consumed by custom mobie Apps (Android, iOS) to allow:

1. **Sellers** to register through App and post product(s) by providing details like Product category, name, quantity, service location radius (Lat/long, distance e.g. 10 KM area).  
1. **Buyer** to get a list of products available locally. Receive notifications about new/updates about local products.

These features mean that the APIs and associated services (PUSH notifications/webhooks/events...) are supported by the backend. Could you please advise on how this requiement can be met with Saleor platform. Thanks !

**Comment by karolkielecki** [comment, marked as answer]
@birender-s Hi there! So are you looking for some kind of multi-vendor feature? If so, then I'm afraid you can't do it. Multi-vendor requires many changes in the code - basically everywhere. Saleor hasn't been designed this way.

**Comment by birender-s** [comment]
hi @karolkielecki , thanks for your response ! Yes, hyperlocal marketplace demands multi-vendor feature as its foundation. If Saleor doesn't support multi-vendor, then yes, expecting hyperlocal marketplace may not be feasible. thanks once again !
